LunrCore/Builder.cs

```diff
@@ -119,6 +119,11 @@ public Builder(params Field[] fields)
         /// The total number of documents indexed.
         /// </summary>
         public int DocumentCount { get; private set; } = 0;
+        
+        /// <summary>
+        /// Gets or sets the separator function used to detect which character should not be part of tokens.
+        /// </summary>
+        public Func<char, bool>? SeparatorFunc { get; set; }
```

Comment:
Ok, and what about `Tokenizer.Tokenize` then? many methods are calling the interface methods and currently it sets the separatorFunc to null.

Reply:
The parameter may be nullable since there's a `??` with the default value when setting the property. To be consistent with the rest of the codebase, declare parameters like this: `Func<char, bool>? separatorFunc = null!` with the bang in the end. That makes it possible to omit the parameter (in which case it will get the default value), but you can't explicitly set it to null.

Reply:
Fixed by 0dfe62a